src/raudio.c

```diff
@@ -419,6 +419,9 @@ void InitAudioDevice(void)
     config.sampleRate = AUDIO_DEVICE_SAMPLE_RATE;
     config.dataCallback = OnSendAudioDataToDevice;
     config.pUserData = NULL;
+#if defined(PLATFORM_WEB)
```

Comment:
I try to avoid `PLATFORM_` flags outside `core.c` module. As `raudio` can be used in standalone mode, it would be better to use `#if defined(__EMSCRIPTEN__)` to avoid `PLATFORM_` flag requirement.

Reply:
added change